auth_token/contrib/rest_framework_auth/serializers.py

```diff
@@ -38,7 +38,7 @@ def validate(self, attrs):
 
 class MobileAuthTokenSerializer(serializers.Serializer):
 
-    uuid = serializers.UUIDField(label=_('device UUID'))
+    uuid = serializers.CharField(label=_('device UUID'), trim_whitespace=True)
```

Comment:
is there any reason for trim_whitespace?

Reply:
it's not required to work but I think it makes sense here. It's actually the default behavior so I removed that.